gulpfile.js

```diff
@@ -0,0 +1,51 @@
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var ts = require('gulp-typescript');
+var del = require('del');
+var webpack = require('webpack');
+
+gulp.task('clean', function(cb) {
+  del('dist/');
+});
+
+gulp.task('ES5', function() {
```

Comment:
This is nitpicking, but I think lowercase would be better for the task name, i.e. `es5`